msgpack/fallback.py

```diff
@@ -259,7 +259,8 @@ def feed(self, next_bytes):
                               DeprecationWarning)
             else:
                 raise
-        assert view.itemsize == 1
+        if view.itemsize != 1:
+            raise ValueError("cannot unpack from multi-byte object")
```

Comment:
I decided to raise a proper exception with explanation here. Multi-byte types are very problematic for fallback code, since there is not way to "cast" ``memoryview`` to a different type.

Usually I'm all for developer responsibility, but honestly, if you're trying to unpack floats, I'm pretty sure you're doing it wrong.